### ISSUE TITLE
Pdf page layout in the two page.

### ISSUE THREAD

**Issue by Sanketjain22** [dompdf/dompdf#2527]
How we can set the page layout from sigle page to two page layout.
During to generate the pdf file, after hit the submit button user can select the option one page layout or two page layout option.
How we can set this in the dompdf.

**Comment by bsweeney** [comment]
You mean to show two pages in the viewer instead of a single page?

**Comment by Sanketjain22** [comment]
Yes, pdf generates the result in the two pages view. In my form, I have an option for a two-page layout or single page layout option before submitting the form. If the user selects the two-page layout so pdf will come in a two-page view.
![image](https://user-images.githubusercontent.com/89734405/131282765-a48cff4e-4de2-4a79-aa6c-7e6e5d97e703.png)

I need to look like this pdf.

**Comment by bsweeney** [comment]
Dompdf and the Cpdf back end do not currently support setting the layout. While it's possible with the PDFLib back end it has to be set through the begin/end document methods, which means you couldn't set it manually.

We'll look at adding this functionality in a future release.

**Comment by Sanketjain22** [comment]
Hello Bsweeney,

Thank you for update.

Regards,
Sanket Jain
